mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -149,7 +149,7 @@ class DeviceTelemetrySerializer(serializers.Serializer):
     )
     charge = serializers.FloatField(
         required=False,
-        source="dn_battery_pct",
+        source="battery_pct",  # use provider API field for consistency
```

Comment:
100% sure?

Reply:
Anyway, I don't get what this `source` is actually doing here. Why don't other fields have it too? This serializer is only used to received data from providers (events or telemetry), not exposing the agency data. @lionel-panhaleux ?

Reply:
It's something that Lionel pointed out. If I've understood correctly, this serializer is what receives data and put it in our base. Right now in base, there are some eventRecords with properties.telemetry field having "dn_batteyr_pct" instead of "battery_pct"

Reply:
You mean this value decides how the property is named in the big `properties` dict? Just like https://github.com/Polyconseil/django-mds/blob/master/mds/factories.py#L673 ?

If so, I'll have to run a script on our instances, and this means it's another issue that deserves its own PR and changelog entry. :stuck_out_tongue: 

And I better understand your comment but it's not accurate. It's not "for consistency", it's because we chose that naming convention in our code.

I'm puzzled because I only knew `source` in the context of a ModelSerializer.

Reply:
Yes, it is as you said. What I wanted to say by "for consistency" is that it would be best if the field named in our properties dict was named the same way as the field we recieve from the provider API. But yeah, it's just a naming convention. However, due to that, we have device telemetries in our base with 2 namings conventions for this battery part